src/test/java/greencity/service/impl/CategoryServiceImplTest.java

```diff
@@ -1,29 +1,34 @@
 package greencity.service.impl;
 
 import static org.junit.Assert.assertEquals;
-import static org.mockito.ArgumentMatchers.any;
-import static org.mockito.ArgumentMatchers.anyLong;
+import static org.mockito.ArgumentMatchers.*;
 import static org.mockito.Mockito.when;
 
-import greencity.GreenCityApplication;
+import greencity.dto.category.CategoryDto;
 import greencity.entity.Category;
 import greencity.entity.Place;
+import greencity.exception.BadCategoryRequestException;
 import greencity.exception.BadRequestException;
 import greencity.exception.NotFoundException;
 import greencity.repository.CategoryRepo;
 import java.util.*;
+import java.util.stream.Collectors;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.mockito.InjectMocks;
 import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
 import org.mockito.junit.MockitoJUnitRunner;
-import org.springframework.boot.test.context.SpringBootTest;
+import org.modelmapper.ModelMapper;
 
-@RunWith(MockitoJUnitRunner.class)
-@SpringBootTest(classes = GreenCityApplication.class)
+@RunWith(MockitoJUnitRunner.Silent.class)
 public class CategoryServiceImplTest {
     @Mock
-    private CategoryRepo categoryRepo;
+    CategoryRepo categoryRepo;
+
+    @Mock
+    ModelMapper modelMapper;
+
     @InjectMocks
     private CategoryServiceImpl categoryService;
```

Comment:
Some fields with default modificator some with private, please decide what is better to use